packages/open-payments/src/client/requests.ts

```diff
@@ -158,23 +158,34 @@ const handleError = async (
   let errorDescription
   let errorStatus
   let validationErrors
+  let errorCode
 
   const { HTTPError } = await import('ky')
 
   if (error instanceof HTTPError) {
-    let responseBody
+    let responseBody:
+      | {
+          error: { description: string; code: string }
```

Comment:
The `responseBody.error?.code` tests below suggest these could be undefined.
```suggestion
          error: { description?: string; code?: string }
```

Reply:
Down below, I'm primarily checking that `responseBody.error` is defined, and if it is, I'm assuming `error.description` and `error.code` are indeed defined.

I added a [commit](https://github.com/interledger/open-payments/pull/484/commits/7a78eb91eb0c232e21aa349c5a84a5b8c99b798b) to simplify the assertion